portability-api/src/main/java/org/datatransferproject/api/ApiExtensionContext.java

```diff
@@ -58,7 +59,7 @@ public ApiExtensionContext(TypeManager typeManager, SettingsExtension settingsEx
   }
 
   @Override
-  public Logger getLogger() {
+  public Layout getLayout() {
```

Comment:
Why do we need to expose this to clients?  Seems like this is an internal implementation detail.